From 7e4b7c4847d98fb1bd12021e1f1627abacc81e1c Mon Sep 17 00:00:00 2001
From: Paul Dettorer Hervot <dettorer@dettorer.net>
Date: Tue, 22 Aug 2023 16:05:02 +0200
Subject: [PATCH] requirements: loosen the dependency version match

---
 requirements.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/requirements.txt b/requirements.txt
index 959b06b..6cb5b37 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,3 +1,3 @@
-osmnx==1.2.2
+osmnx>=1.2.2
-Shapely>=1.8,<2.0
+Shapely>=1.8
-pyyaml==6.0.0
\ No newline at end of file
+pyyaml>=6.0.0
-- 
2.41.0

